src/NexusMods.CrossPlatform/Process/IProcessFactory.cs

```diff
@@ -10,9 +10,7 @@ public interface IProcessFactory
     /// <summary>
     /// Executes the given command that starts the process.
     /// </summary>
-    /// <param name="command">The command to execute.</param>
-    /// <param name="cancellationToken">Allows you to cancel the task, killing the process prematurely.</param>
```

Comment:
You are losing the info about CancelationToken killing the process, which isn't a given.